go/arrow/array/struct.go

```diff
@@ -81,6 +81,15 @@ func NewStructData(data arrow.ArrayData) *Struct {
 func (a *Struct) NumField() int           { return len(a.fields) }
 func (a *Struct) Field(i int) arrow.Array { return a.fields[i] }
 
+func (a *Struct) ValueStr(i int) string {
```

Comment:
add a docstring comment on here to indicate that the value is returned as a JSON string please.